### PR TITLE
Blogbuild

### PR DESCRIPTION
**Pull request by rmjjjt**
First test of the blog sections added

### CODE REVIEW

File: css/images.css [hunk 1]
```diff
@@ -46,5 +46,16 @@ body {
...
+	width: 100px;
+	height: 100px;
 	float: left;
+}
+
+.blogContainer img {
+	width: 100px;
+	height: 100px;
+	float: left;
+	padding-right: 10px;
```
Comment (JessI23) [inline css/images.css]: since you have padding on 3 sides i would combine them into a single padding property

File: blogindex.php [hunk 2]
```diff
@@ -52,21 +52,27 @@
...
-				<a target=_blank href="#">date/title</a>
-				<p>This is a summary of a blog post. It is 131 characters long. Probably this many more should do it - don’t want too much stuff here! <a href="#">...read more</a></p>
-			</div>
-			<div class="blogSection">
-				<a target=_blank href="#"><img src="images/logo.png"></a>
-				<a target=_blank href="#">date/title</a>
-				<p>This is a summary of a blog post. It is 131 characters long. Probably this many more should do it - don’t want too much stuff here! <a href="#">...read more</a></p>
-			</div>
+			<?php
+				require('classes.php');
```
Comment (JessI23) [inline blogindex.php]: php file only contains one class, maybe rename to blogpost.php?

File: blogindex.php [hunk 3]
```diff
@@ -52,21 +52,27 @@
...
+
+//				foreach($arrayReverse as $key => $value) {
+//					echo '<div class="blogPost"><a href=blogarticles.php?blog=' . $key . '>' . $value->getTitle() . '</a><br><p>' . $value->getSummary() . '</p>';
+//				}
+				$topThree = array_slice($arrayReverse, 0, 3 , true);
+
+				foreach($topThree as $key => $value) {
```
Comment (JessI23) [inline blogindex.php]: maybe rename $value to  e.g. $blog or $blogPost to make it easier to follow?

File: blogindex.php (hunk 2 above)
Comment (rmjjjt) [inline blogindex.php]: Fixed in latest commit - the red lines are wrong (removed as were only a template).

File: blogindex.php (hunk 3 above)
Comment (rmjjjt) [inline blogindex.php]: Changed in blogpost.php class commit

File: blogarticles.php [hunk 4]
```diff
@@ -50,16 +50,30 @@
...
-			<p>Aenean leo metus, vehicula id venenatis sed, aliquet in nunc. Curabitur laoreet eget metus in efficitur. Suspendisse in turpis id velit consequat maximus eget hendrerit magna. Curabitur eget faucibus felis. Mauris gravida egestas leo, non sollicitudin elit viverra et. Aenean ut erat viverra elit fringilla laoreet vitae sit amet nisl. Fusce posuere pulvinar urna, sed mattis ligula eleifend in. Nulla mattis nibh quis eleifend finibus. Nam sollicitudin tempus quam, tristique suscipit turpis ullamcorper et. In nec commodo ex. Vivamus venenatis eu nisl eget consequat. In vehicula, magna sit amet venenatis vehicula, turpis neque semper ligula, sed dapibus sapien diam sagittis tortor.</p>
-		</div>
+			<?php
+
+			// use $blog as key to get body text from arrays
+			foreach($blog as $key => $value) {
+				echo '<img src="' . $value->getImage() . '">' . $value->getBody();
```
Comment (apgmartin) [inline blogarticles.php]: Need an image alt

File: blogindex.php [hunk 5]
```diff
@@ -53,25 +53,13 @@
...
-//				foreach($arrayReverse as $key => $value) {
-//					echo '<div class="blogPost"><a href=blogarticles.php?blog=' . $key . '>' . $value->getTitle() . '</a><br><p>' . $value->getSummary() . '</p>';
-//				}
-				$topThree = array_slice($arrayReverse, 0, 3 , true);
+				// connect to class sheet
+				require('includes/classes/blogarrays.php');
 
+				// setting url, extracting the header and summary text for displaying on the blog home page
 				foreach($topThree as $key => $value) {
 					echo '<div class="blogPost"><a href=blogarticles.php?blog=' . $key . '>' . $value->getTitle() . '</a><br><img src="' . $value->getImage() . '"><p>' . $value->getSummary() . '</p>';
```
Comment (apgmartin) [inline blogindex.php]: Need an image alt

**Pull request merged**